### ISSUE TITLE
Endorsement section on landing page

### ISSUE THREAD

**Issue by dawranliou** [coast-framework/docs#1]
I'd like to propose a feature:

1. Add an endorsement table in the db. The entries can be as simple as a URL to the twitter status.
1. Add Admin pages to CRUD the endorsement entries.
1. Render several entries on the landing page.

At the very least, I think we can have @swlkr, @elamje, and @dawran6 to endorse Coast.

What are your thoughts?

**Comment by swlkr** [comment]
Oh this is a good idea! Could we embed tweets too?

**Comment by dawranliou** [comment]
Yes, from the [Twitter Publish](https://publish.twitter.com/#), the embed tweet would look like this:

```html
<blockquote class="twitter-tweet">
  <p lang="en" dir="ltr">Scheer looks like an overgrown toddler stuffed in a suit. It&#39;s largely the vacant look on
    his face like that of a child that hasn&#39;t yet learned to form emotions.</p>&mdash; Dmitri Sotnikov ☭⚛ (@yogthos)
  <a href="https://twitter.com/yogthos/status/1180523442711547909?ref_src=twsrc%5Etfw">October 5, 2019</a>
</blockquote>
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```

So I guess we can put these in the db:

1. URL
1. Body
1. Twitter user name
1. Date

The alternative is to hardcoded the endorsement tweets in the view instead of putting them in the db.
